scripts/jenkins/qat/checks.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Checks if all pre-requirements for qat-dpdk are up.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+sudo dmesg | grep -i qat
+
+HPAGES=$(sudo cat /proc/cmdline | grep "hugepages=1024")
```

Comment:
The app deployment has a request for HugePage resources. The deployment test would fail if the cluster cannot meet that requirement so in that sense this check is not necessary. Moreover, this would also fail if the setup has HugePages but there are more/less of them configured. Let's leave this out.